src/arm/96boards.c

```diff
@@ -36,6 +36,7 @@
 
 #define DT_BASE "/proc/device-tree"
 
+#define PLATFORM_NAME_ROCK960 "ROCK960"
```

Comment:
Please keep this list sorted alphabetically. Even though it is not sorted at the moment we can do the cleanup anytime.